File: Lab_C#/Asignacion_01(While)/Asignacion01(While).cs
using System;

class Program
{
    static void Main(string[] args)
    { 
        string nombre;
        string apellido;
        int num_veces = 1; //contador que inicia en 1

        Console.Write("Introduce tu nombre: "); //solicitar al usuario que introduzca su nombre
        nombre = Console.ReadLine();

        Console.Write("Introduce tu apellido: "); //solicitar al usuario que introduzca su apellido
        apellido = Console.ReadLine();

        while (num_veces <= 10) //bucle while que se ejecuta mientras el numero sea menor o igual a 10
        {
            Console.WriteLine("Hola " + nombre + " " + apellido); //mensaje de salida con el nombre y apellido
            num_veces++; // incrementacion del contador en 1 veces
        }

    }
}

File: Lab_C++/Modulo_5/Asignacion_01(While).cpp
#include <iostream>
using namespace std;

int main()
{
    string nombre;
    string apellido;
    int num_veces = 1; //contador que inicia en 1

    cout << "Introduce tu nombre: "; //solicitar al usuario que introduzca su nombre
    cin >> nombre;

    cout << "Introduce tu apellido: "; // solicitar al usuario que introduzca su apellido
    cin >> apellido;

    while (num_veces <= 10) //bucle while que se ejecuta mientras el numero sea menor o igual a 10
    {
        cout << "Hola " << nombre << " " << apellido << endl; //mensaje de salida con el nombre y apellido
        num_veces = num_veces + 1; // incrementacion del contador en 1 veces
    }
    return 0;
}